<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QTextBrowser" name="browser"/>
    </item>
    <item row="0" column="4" colspan="6">
     <widget class="Phonon::VideoPlayer" name="player"/>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QLineEdit" name="lineedit"/>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="time">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8" colspan="2">
     <widget class="Phonon::VolumeSlider" name="playerVolume"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="connectButton">
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="syncButton">
      <property name="text">
       <string>SYNC</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QCheckBox" name="random">
      <property name="text">
       <string>Random</string>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QCheckBox" name="loop">
      <property name="text">
       <string>Loop</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="5" colspan="3">
     <widget class="Phonon::SeekSlider" name="playerSeek"/>
    </item>
    <item row="2" column="7">
     <widget class="QPushButton" name="prev">
      <property name="text">
       <string>Prev</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="playPause">
      <property name="text">
       <string>Play/Pause</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="next">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="stop">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="fullScr">
      <property name="text">
       <string>Full screen</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="10">
     <widget class="QTreeWidget" name="tree">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
